.github/workflows/rule-tester.yml

```diff
@@ -1,34 +1,64 @@
 name: Deploy Rule Tester Endpoint
+
 on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      branch:
+        required: false
+        type: string
+
```

Comment:
Branch should not be an input param. The checkout action will use the current branch by default